packages/cashier/src/pages/account-transfer/account-transfer-form/account-transfer-form.tsx

```diff
@@ -467,26 +584,30 @@ const AccountTransferForm = ({
                 {({ errors, handleChange, isSubmitting, touched, setFieldValue, setFieldTouched, setFieldError }) => (
                     <React.Fragment>
                         {isSubmitting || accounts_list.length === 0 ? (
-                            <div className='cashier__loader-wrapper'>
+                            <div className='cashier__loader-wrapper' data-testid='cashier__loader-wrapper'>
```

Comment:
Could you follow our `data-testid` naming convention in the bellow file and update all the data-testids
https://wikijs.deriv.cloud/Frontend/General-information/test_guide

Reply:
Make all the testids match the naming convention.